apps/financial-aid/api/src/app/modules/application/models/updateApplication.response.ts

```diff
@@ -0,0 +1,15 @@
+import { UpdateApplicationResponse } from '@island.is/financial-aid/shared/lib'
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+
+import { ApplicationModel } from './application.model'
+import { ApplicationFiltersModel } from './applicationFilter.model'
+
+@ObjectType()
+export class UpdateApplicationResponseModel
```

Comment:
Should this be named `Model`? Aren't we using that term for models in the db?

Reply:
Reason I name dit mode is because the shared type is UpdateApplicationResponse, so i needed to know the difference 🙈  Have any name suggestions?

Reply:
The type one is UpdateApplicationResponseType and this one is UpdateApplicationResponse?